docker-compose.full.yml

```diff
@@ -64,6 +62,12 @@ services:
       service: flower
     links:
       - mq
+  kibana:
+    extends:
+      file: docker-services.yml
+      service: kibana
+    links:
+      - es
```

Comment:
Accidental transfer?

Reply:
No, this was moved out of the 'base services' section into the 'monitoring' section to be more consistent